lib/super_digit.rb

```diff
@@ -1,15 +1,30 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n) where n is the length of the number passed to the method
+# Space Complexity - O(n) due to array of digits, .sum
 def super_digit(n)
```

Comment:
This works, but the time complexity is O(d^2) where d is the number of digits because the summation is O(d) and you're doing it inside the recursive method (going d times).